www/source/partials/docs/_dev-pkgs-write-plans.html.md.erb

```diff
@@ -102,7 +102,7 @@ The following sections describe each of these steps in more detail.
 #### Create your package identifier
 The origin is a place for you to set default privacy rules, store your packages, and collaborate with teammates. For example, the "core" origin is where the core maintainers of Habitat share packages that are foundational to building other packages. If you would like to browse them, they are located in the [core-plans repo](https://github.com/habitat-sh/core-plans), and on [Habitat Builder's Core Origin](https://bldr.habitat.sh/#/pkgs/core).
 
-Creating packages for a specific origin requires that you have access to the secret key for that origin. The secret key will be used to sign the package when it is built by the hab-plan-build command. Keys are kept in `$HOME/.hab/cache/keys` on the host machine and `/hab/cache/keys` while in the studio. For more information on keys, see [Keys](/docs/glossary/#glossary-keys).
+Creating packages for a specific origin requires that you have access to the that origin's private key. The private origin key will be used to sign the package when it is built by the hab-plan-build command. Origin keys are kept in `$HOME/.hab/cache/keys` on the host machine and `/hab/cache/keys` while in the studio. For more information on origin keys, see [Keys](/docs/glossary/#glossary-keys).
```

Comment:
Again, path consistency @mwrock ?

Reply:
I'll add Windows docs as I work through the Windows guides.